test/language/expressions/generators/name-property-descriptor.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Generator functions should define a `name` property.
+includes: [propertyHelper.js]
+es6id: 25.2.4
+---*/
+
+var g = function*() {};
+
+verifyNotEnumerable(g, 'name');
+verifyNotWritable(g, 'name');
+verifyConfigurable(g, 'name');
```

Comment:
There's a separate test for the value of the `name` property in `master` already: https://github.com/tc39/test262/blob/c49e2bd6b902aa699bedeabcf6e84adf99688a99/test/language/expressions/generators/name.js

@bterlson Do you think it's worthwhile to keep these assertions separate? Or should I merge the existing file into this one?